packages/react-components/react-breadcrumb-preview/stories/Breadcrumb/BreadcrumbWithTooltip.stories.tsx

```diff
@@ -182,9 +182,7 @@ const getTooltipContent = (breadcrumbItems: readonly Item[] | undefined) => {
     return (
       <>
         {acc}
-        {arr[0].item !== initialValue.item && (
-          <BreadcrumbDivider style={{ display: 'inline', verticalAlign: 'middle' }} />
-        )}
+        {arr[0].item !== initialValue.item && ' > '}
```

Comment:
wondering how will this resolve on screen readers ? as before it would announce the `li` but now suddenly there is no html element in the order flow ?

Reply:
That's the thing, it was a mistake to have it here as `li` element. As this is a tooltip content, it has to be just a string

Reply:
gotcha, it wasn't clear to me from the description. ty